instrumentation/servlet/servlet-5.0/javaagent/src/test/groovy/JettyServlet5MappingTest.groovy

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import jakarta.servlet.Servlet
+import jakarta.servlet.ServletException
+import jakarta.servlet.http.HttpServlet
+import jakarta.servlet.http.HttpServletRequest
+import jakarta.servlet.http.HttpServletResponse
+import org.eclipse.jetty.server.Server
+import org.eclipse.jetty.servlet.ServletContextHandler
+import spock.lang.IgnoreIf
+
+@IgnoreIf({ !jvm.java11Compatible })
+class JettyServlet5MappingTest extends AbstractServlet5MappingTest<Object, Object> {
```

Comment:
Jetty 11 classes are compiled with java 11 so need to disable test on java 8. As class is poked with reflection before `IgnoreIf` applies can't use java 11 classes in method signatures because of `UnsupportedClassVersionError`.